text_collector_examples/mellanox_hca_temp

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -eu
+
+# Script to read Mellanox HCA temperature using the Mellanox mget_temp_ext tool
+
+# Copyright 2018 Jan Phillip Greimann <jan.greimann@profitbricks.com>
```

Comment:
Apache2 is fine, but we require code in our repos to be set to `Copyright 2018 The Prometheus Authors`.

You're welcome to add an `Author: Jan Phillip Greimann <jan.greimann@profitbricks.com>` below the copyright.